services/QuillLMS/client/app/bundles/Proofreader/components/proofreaderActivities/container.tsx

```diff
@@ -610,19 +614,19 @@ export class PlayProofreaderContainer extends React.Component<PlayProofreaderCon
       const { edits, necessaryEdits, loadingFirebaseSession, showWelcomePage, } = this.state
       const { currentActivity } = proofreaderActivities
 
-      if (loadingFirebaseSession) { return <LoadingSpinner />}
+      if (loadingFirebaseSession) { return <div className="passage-container" ref={this.passageContainer}><LoadingSpinner /></div>}
 
       if (showWelcomePage) { return <WelcomePage onNextClick={this.handleNextClick} /> }
 
-      if (session.error) { return <div>{session.error}</div> }
+      if (session.error) { return <div className="passage-container" ref={this.passageContainer}>{session.error}</div> }
 
-      if (!currentActivity) { return <LoadingSpinner />}
+      if (!currentActivity) { return <div className="passage-container" ref={this.passageContainer}><LoadingSpinner /></div>}
```

Comment:
I feel like we should just store this in a variable since we're passing it twice